### ISSUE TITLE
Scroll container key prop

### ISSUE THREAD

**Issue by martincrisdarr** [1000ship/react-scroll-motion#35]
Hi,  the ScrollContainer component print me a key prop error in console and i don´t know how to fix it. "Each child in a list should have a unique "key" prop."

**Comment by omarr45** [comment]
Any updates on how this could be fixed?

**Comment by swbatallas** [comment]
Any updates on how this could be fixed? Again :)

**Comment by won-h** [comment]
help

**Comment by masafumimori** [comment]
I don't think you can fix this on your code basis. It is rather an issue of the library.
This PR should fix the issue as it includes [this commit](https://github.com/1000ship/react-scroll-motion/pull/37/commits/054b587a0d85c964a18347b949efa9dc26df0cfd).
